packages/lesswrong/components/search/ExpandedCommentsSearchHit.tsx

```diff
@@ -0,0 +1,127 @@
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import { Link } from '../../lib/reactRouterWrapper';
+import React from 'react';
+import type { Hit } from 'react-instantsearch-core';
+import { Snippet } from 'react-instantsearch-dom';
+import { postGetPageUrl } from '../../lib/collections/posts/helpers';
+import { tagGetCommentLink } from '../../lib/collections/tags/helpers';
+import { TagCommentType } from '../../lib/collections/comments/types';
+import TagIcon from '@material-ui/icons/LocalOffer';
+import { userGetProfileUrlFromSlug } from '../../lib/collections/users/helpers';
+// We're not using Link, just useHistory
+// eslint-disable-next-line no-restricted-imports
+import { useHistory } from 'react-router-dom';
```

Comment:
I think we can get `history` for the purpose of `history.push` from our custom `useNavigation` hook